docs/docusaurus/docs/cloud/quickstarts/python_api_quickstart.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Python scripts'
+title: 'Quickstart for GX Cloud and Python scripts'
+id: python_quickstart
+description: Connect to a GX Cloud account from a Python script.
+---
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+import PrereqPython from '/docs/components/prerequisites/_python_version.md'
+import SetupAndInstallGx from '/docs/components/setup/link_lists/_setup_and_install_gx.md'
+import DataContextInitializeInstantiateSave from '/docs/components/setup/link_lists/_data_context_initialize_instatiate_save.md'
+
+In this quickstart, you'll learn how to use GX Cloud from a Python script or interpreter, such as a Jupyter Notebook. You'll install Great Expectations, configure your GX Cloud environment variables, connect to sample data, build your first Expectation, validate data, and review the validation results through Python code.
+
+## Data validation workflow
+
+The following diagram illustrates the end-to-end GX data validation workflow that you'll implement with this quickstart. Click a workflow step to view the related content.
+
+```mermaid
+flowchart LR
+%%{init: {"themeVariables": {"fontSize" : "24px"}}}%%
+
+    1(Install\n<b>GX</b>) --> 2("Create a\n<b>Data Context</b>")
+
+    2 --> 3
+
+    3("Connect\nto data</b>")
+
+    3 --> 4
+
+    4("Create a\n<b>Validator</b>") --> 5("Create\n<b>Expectations</b>")
+
+    5 --> 6
+
+    6("Run a\n<b>Checkpoint</b>") --> 7("View\n<b>Validation Results</b>")
+
+%% Link each workflow step to section on page.
+click 1 "#install-gx"
+click 2 "#create-a-data-context"
+click 3 "#connect-to-data"
+click 4 "#connect-to-data"
+click 5 "#create-expectations"
+click 6 "#validate-data"
+click 7 "#validate-data"
+```
+
+## Prerequisites
+
+- <PrereqPython />
+- pip
+- An internet browser
+- A GX Cloud Beta Account
```

Comment:
```suggestion
- A GX Cloud Beta Account
```
Link to the GX Cloud Beta [sign-up page](https://greatexpectations.io/cloud), or the existing procedure.

Reply:
In an upcoming commit: Updated this and the internet access/browser prerequisite to be in line with the existing GX Cloud docs.